webpack.config.js

```diff
@@ -3,6 +3,16 @@ const { resolve } = require('path');
 const webpack = require('webpack');
 
 module.exports = {
+  context: resolve(__dirname, 'src'),
+  devtool: 'cheap-module-eval-source-map',
+  devServer: {
+    hot: true,
+    contentBase: resolve(__dirname, 'dist'),
+    publicPath: '/'
+  },
+  resolve: {
+    extensions: ['.js', '.scss'],
+  },
```

Comment:
I'm not questioning, I'm just curious, what is your order based on? I can't see through it.